gbus/worker.go

```diff
@@ -391,10 +393,15 @@ func (worker *worker) invokeHandlers(sctx context.Context, handlers []MessageHan
 		return nil
 	}
 
-	//retry for MaxRetryCount, back off by a Fibonacci series 50, 50, 100, 150, 250 ms
+	//retry for MaxRetryCount, back off by a jittered stratergy
+	seed := time.Now().UnixNano()
+	random := rand.New(rand.NewSource(seed))
 	return retry.Retry(action,
 		strategy.Limit(MaxRetryCount),
-		strategy.Backoff(backoff.Fibonacci(50*time.Millisecond)))
+		strategy.BackoffWithJitter(
+			backoff.BinaryExponential(10*time.Millisecond),
```

Comment:
lets have these values exposed via configuration with default values

Reply:
just the `10*time.Millisecond`, right?